texastribune/donations
# Remove font-awesome and replace with svgs

#### What's this PR do?

Swaps font-awesome spans with our `c-icon` svg markup

#### Why are we doing this? How does it help us?

Icon fonts have some perf implications and we're moving to SVGs in our other repos.

#### How should this be manually tested?

`make`
`yarn run dev`
`make restart` for good measure

- Check the footer social links
- Check the /circle page stars
- Make sure nothing barks at you in the build

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

I'm using a macro. I don't see those used anywhere else so if we're not into macros, I'm happy to change. It felt like a lot to use [jinja's `with` syntax](http://jinja.pocoo.org/docs/2.9/templates/#with-statement), so a macro _looked_ more concise. I'm just not sure if there's some negative impact of using macros and that's why we haven't implemented.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/1781295778

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:

* [ ] *Remove _fontawesome.scss file (keeping it there for now just in case we find we need it in the short term)*
